### DISCUSSION TITLE
0002 - MyST Core Design Philosophy

### DISCUSSION THREAD

**Discussion by chrisjsewell** [jupyter-book/myst-enhancement-proposals#4] [MEP]
Lets start discussing! https://github.com/executablebooks/myst-eps/pull/5

**Comment by chrisjsewell** [comment]
pinging @rowanc1 @choldgraf @mmcky etc
this is intended to be a document on some initial high-level goals for MyST, that I'd definitely like your input on 😄 

Obviously, some of this can be pulled from https://spec.myst.tools/spec

and the you'd have this higherarchy of `mep -> spec -> sphinx paser, js parser, ...`

**Reply by rowanc1** [reply]
Hack MD document is here:
https://hackmd.io/LOdkezLAS-6Wrq4HoHJ5zQ

**Reply by chrisjsewell** [reply]
cheers @rowanc1
